src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -382,6 +382,164 @@ public new abstract class Arm64 : AdvSimd.Arm64
         public static unsafe Vector<ulong> AndAcross(Vector<ulong> value) => AndAcross(value);
 
 
+        ///  BitwiseClear : Bitwise clear
+
+        /// <summary>
+        /// svuint8_t svbic[_u8]_m(svbool_t pg, svuint8_t op1, svuint8_t op2)
+        /// svuint8_t svbic[_u8]_x(svbool_t pg, svuint8_t op1, svuint8_t op2)
+        /// svuint8_t svbic[_u8]_z(svbool_t pg, svuint8_t op1, svuint8_t op2)
+        ///   BIC Ztied1.B, Pg/M, Ztied1.B, Zop2.B
+        ///   BIC Zresult.D, Zop1.D, Zop2.D
+        /// svbool_t svbic[_b]_z(svbool_t pg, svbool_t op1, svbool_t op2)
+        ///   BIC Presult.B, Pg/Z, Pop1.B, Pop2.B
```

Comment:
I don't think we are supporting the `BIC Presult.B, Pg/Z, Pop1.B, Pop2.B`. Can we remove it from the summary docs in this and other APIs?

Reply:
`Sve. BitwiseClear(mask)` will convert the mask to a normal vector and use the vector BIC instruction. That's not great. We should be using the mask variant where possible. We need something in lowering? hwinstrinsics? to spot this.

I've raised an issue to cover it: https://github.com/dotnet/runtime/issues/101970

Is it ok to leave the summary as is, because we'll have to restore them when fixing the issue. Plus it makes it easier to spot them when we fix it.

Reply:
sure